extensions/wikia/AdEngine/js/spec/provider/gpt/adDetect.spec.js

```diff
@@ -21,12 +25,19 @@ describe('Method ext.wikia.adEngine.provider.gpt.adDetect.onAdLoad', function ()
 				}
 			};
 
-			gptHop = modules['ext.wikia.adEngine.provider.gpt.adDetect'](mocks.log, mocks.window, mocks.adContext);
+			mocks.iframe.contentWindow = {
+				'AdEngine_adType': forceAdType,
```

Comment:
What do you think about setting it to default `'unknown'` if it's not passed within `desktop()` parameters?